File: ASP.NET CMS Administration Template User Administration/[VB]-ASP.NET CMS Administration Template User Administration/VB/WebApplication1.ConvertedToVBNet/Admin/admin.Master.vb
Imports System.Collections.Generic
Imports System.Linq
Imports System.Web
Imports System.Web.UI
Imports System.Web.UI.WebControls
Imports System.Security.Permissions

Namespace Admin
	<PrincipalPermission(SecurityAction.Demand, Authenticated := True, Role := "Super Administrator")> _
	Public Partial Class admin
		Inherits System.Web.UI.MasterPage

		Protected Sub Page_Load(sender As Object, e As EventArgs)

		End Sub
	End Class
End Namespace

File: ASP.NET Great Holding Page (Coming Soon  under construction pages)/[C#]-ASP.NET Great Holding Page (Coming Soon  under construction pages)/C#/WebRole1/Admin/admin.Master.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.UI;
using System.Web.UI.WebControls;
using System.Security.Permissions;

namespace WebApplication1.Admin
{
    [PrincipalPermission(SecurityAction.Demand, Authenticated = true, Role = "Super Administrator")] 
    public partial class admin : System.Web.UI.MasterPage
    {

               protected void Page_Load(object sender, EventArgs e)
        {
            
        }
    }
}